### PR TITLE
Update source of total aptc for calculation

### PR DESCRIPTION
**Pull request by RyanEddyIC**
RM-94744

### CODE REVIEW

**Review by markgoho — COMMENTED** [review]
@RyanEddyIC if this is a bug fix, where is the failing test that has now been fixed with this change in code?

**Comment by RyanEddyIC** [comment]
@markgoho there was no failing test.

**Comment by markgoho** [comment]
Yes, so let's write one 🤓

File: app/models/insured/factories/self_service_factory.rb [hunk 1]
```diff
@@ -68,14 +68,15 @@ def self.update_enrollment_for_apcts(reinstatement, applied_aptc_amount)
...
         max_applicable_aptc = if EnrollRegistry[:apply_aggregate_to_enrollment].feature.is_enabled
                                 applicable_aptc_by_member.values.sum
                               else
+                                eli_fac_obj = ::Factories::EligibilityFactory.new(reinstatement.id, reinstatement.effective_on)
                                 eli_fac_obj.fetch_max_aptc
                               end
 
-        reinstatement.update_attributes!(elected_aptc_pct: (applied_aptc_amount / max_applicable_aptc), applied_aptc_amount: cost_decorator.total_aptc_amount, aggregate_aptc_amount: max_applicable_aptc)
+        cd_total_aptc = cost_decorator.total_aptc_amount
+        reinstatement.update_attributes!(elected_aptc_pct: (cd_total_aptc / max_applicable_aptc), applied_aptc_amount: cd_total_aptc, aggregate_aptc_amount: max_applicable_aptc)
```
Comment (ymhari) [inline app/models/insured/factories/self_service_factory.rb:79]: Add test cases where 
1. The aptc amount for the family is greater than the product premium. 
2. The aptc amount is less than the product premium.
3. And pct value should ignore ehb of product, so when the an elected aptc is greater than enrollment premium, applied aptc amount will be less than 100% however we still need the pct as 1.

**Review by ymhari — CHANGES_REQUESTED** [review]

**Review by ymhari — APPROVED** [review]
Fixes pct ratio calculation during apply aggregate.

**Pull request merged**